Serb Buffs (Or so I fukken THOUGHT)

Mycah142 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

1. Increased combat stats to just below IHC round start.

2. Added Cybernetic Installer implanter subtype. Also created Disposable subsubtype.

3. Cleared up description on subdermal armor, infantrymen ration packs, and implanter, to be easier to identify utility.

4. Created and mapped accessory box, containing uniform armors and concealed holsters, mapped in.

5. Improved fire threshhold of SCAF to half a firesuit, instead of a sixth (default space-suit value).

6. Mapped in missing serb shotgun crate (Valo).

7. Updated medical pack to match other available medical packs with meralyne.

8. Added Subdermal Armor and Leg Muscle single-use cybernetic installer to the uplink/implants list.

9. Created and mapped cybernetic applicator box, containing 6 subdermal armors and enough leg muscles for everyone.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

 - - Serbs are the closest antag Eris has to "round ending" threat levels, and yet because they spawn usually an hour or so into a round, they're dunked on more often than not. They can't take a hit, can't give a hit. They have a SUPERB voidsuit and weaponry, but because some grab-fu or a single crack grenade (or their own WP grenade) sets them into paincrit very quickly, they're essentially just walking loot bags for players to easily outmatch thanks to the length of the round and general items available to the crew. Three serbs getting dunked on by a single doctor with a hypersaw, on a regular basis, is no longer "skill issue", it's a design problem.

 - The stat improvement is just so that they're at least SOMEWHAT close to a **round start** IHC. This isn't unreasonable.

 - The Cybernetic Installer will be in the Uplink, as well as pre-boxed for Serbs. There's a small selection of cybernetics offered in the Uplink, but because the Serb Bio/Mec stats are so poor, they can't really utilize these reasonably. The Installer should make it far simpler to ready up. This being on the Uplink also means that your bog standard contractor also has another leg up.

 - Descriptions on some items were unclear, like on MRE packages say "medicinal properties for field work", which isn't really clear on what medicines, so a player without foreknowledge wouldn't know that it's painkillers, which would heavily assist the Serbs take more than a good knock to the chin by Vagabond Lou Ferrigno. Subdermal armor also only implants to the chest, the description wasn't clear on this.

 - Another thing Serbs lack is real armor stacking. Crew can stack armor for days, a TE can out-stack the SCAF with just a single armor part and duct tape, and be just as fast in his advanced suit. So, I made a new box that contains uniform armors. The Concealed Holsters are for convenience, as it's a LOADOUT ITEM, and isn't unreasonable for sidearm carry.

 - Serbs now have White Phos grenades. Dozens of them. Dying to your own WP grenade is cringe. So I increased the fire threshold to at least not enter paincrit immediately.

 - Valo mentioned he wasn't able to map in the serb shotgun crate he made previously, and asked I add it as well. Enjoy the Bojvic.

 - Serb specific medicines kit still had Bicaridine, when others were updated with meralyne instead, this was missed.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Testing
Cybernetic Installer:
[dreamseeker_93zLESUFTK.webm](https://github.com/discordia-space/CEV-Eris/assets/11076040/4edc0fda-6522-4261-9cc0-61980be9738b) The item in action, a video.

Disposable Cybernetic Installer:
[dreamseeker_I5M9gPi294.webm](https://github.com/discordia-space/CEV-Eris/assets/11076040/d49b4145-b2ff-4e50-a21e-7d98d1c059c1) A video

![dreamseeker_wzapJox3b9](https://github.com/discordia-space/CEV-Eris/assets/11076040/dba6263f-1c43-403a-a41a-9c97bc52d06d) Now with visually distinct sprites.


Mapped accessory box:
![dreamseeker_iKHkz7FSvx](https://github.com/discordia-space/CEV-Eris/assets/11076040/302cc2ae-c964-4f93-b633-b8cb8ad9c1c5)

Shotgun Crate:
![dreamseeker_XfktyaVc3r](https://github.com/discordia-space/CEV-Eris/assets/11076040/dce3f771-8e6d-42b6-a3e2-0751fe0f7cb6)

Descriptive Text for MRE items:
![dreamseeker_W8oNGwwI0O](https://github.com/discordia-space/CEV-Eris/assets/11076040/6b947ef6-4b73-4389-ba8b-e345ec159646)


<!-- Describe the tests you ran with your addition. It is recommended to add images, videos and step-by-step explanations of conducted testing. -->

## Changelog
:cl:
add: Cybernetic Installer to Uplink
add: Cybernetic Installer Kit to Serb Base
add: Uniform Accessory Kit for uniform armors and concealed holsters (Serb Only)
add: Missing Serb Shotgun crate missed in a previous PR
tweak: Serb stats slightly hire, closer to IHC round start values
tweak: Subdermal Armor Description to accurately represent it as a torso only cybernetic
tweak: Serb MRE Description to better represent medicinal properties
tweak: SCAF armor fire damage threshold to be half that of the firesuit
imageadd: Cybernetic Installer has it's own sprite, thanks to SirRichardFrancis (chad only)
imageadd: Cybernetic Installer cybernetic-unique sprites, again thank you SirRichardFrancis
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Firefox13 commented on code/game/objects/items/devices/organ_module/passive/armor.dm:
```diff
@@ -1,6 +1,6 @@
 /obj/item/organ_module/armor
 	name = "subdermal armor"
-	desc = "A set of subdermal steel plates, designed to provide additional impact protection while remaining lightweight."
+	desc = "A set of subdermal steel plates, designed to provide additional impact protection to the torso while remaining lightweight."
```
It actually works by reducing all brute damage to THE WHOLE BODY, but it only works if it's installed in the torso.

Firefox13 reviewed (commented):
I'm all in for cybernetics installers only if they're one-use (can't put something in them) (teamwork etc etc)

Firefox13 commented:
Just give them several installers. I'm all in for even contractors getting these, but this means all of them should come pre-installed with cybernetics. Again, I like this idea, but as long as they're one-use, kind of like /tg/ CMO HUD implanter that they spawn with a locker.

Mycah142 commented:
Do you have any other change requests? Just so I can settle it in one more commit... Sometime. I ain't got time today.

Mycah142 commented:
Found a bug that let the Cybernetic Applicator also do implants. Bug, fixed. Now has disposal variant, which should be the only thing players will encounter.

Made a deferred kit for cybernetics for serbs, enough for six. No arm blades, just subdermals and leg muscles.

Reworked armblade uplink item to be applicator now. Always bugged me the armblade was just the fukken item I had to install myself.

Included instructions in the uplink near the other cybernetics on how to add new ones.

Should be good to go.

hyperioo commented on code/game/objects/items/devices/organ_module/_module.dm:
```diff
@@ -24,3 +25,20 @@
+
+	if(istype(I, /obj/item/implanter))
+		to_chat(user, SPAN_NOTICE("You cannot insert cybernetics into an implant applicator."))
+		return
```
```suggestion
		return

```

hyperioo commented on code/game/objects/items/weapons/implant/implanter.dm:
```diff
@@ -78,3 +79,89 @@
+	return
+
+/obj/item/implanter/installer/update_icon()
+	if(mod)
+		icon_state = mod.mod_overlay
+	if(mod.mod_overlay == null)
+		icon_state = "installer_full"
+	if(!mod)
+		icon_state = "installer_empty"
```
```suggestion
	if(mod)
		if(mod.mod_overlay == null)
			icon_state = "installer_full"
		else
			icon_state = mod.mod_overlay
	else
		icon_state = "installer_empty"
```

hyperioo replied:
Otherwise it will runtime when trying to check `if(mod.mod_overlay == null)` when mod is null

hyperioo reviewed (commented):
^
